tensorboard/version.py

```diff
@@ -15,7 +15,7 @@
 
 """Contains the version string."""
 
-VERSION = "2.17.0a0"
+VERSION = "2.18.0a0"
```

Comment:
Just a qq - is this for the next release?

Reply:
Yes, our nightly releases are produced from the master branch and use this number, which is a higher version from the latest stable release. See for example our release history: https://pypi.org/project/tb-nightly/#history vs https://pypi.org/project/tensorboard/#history.

Now that we released version 2.17.0, our nightlies would start using the 2.18.x versions.